packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -235,9 +236,7 @@ export class PersonalizeMiddleware {
     // Execute targeted experience in CDP
     const { ua } = userAgent(req);
     const params = this.getExperienceParams(req);
-    const pointOfSale = site.pointOfSale
-      ? site.pointOfSale[language] || site.pointOfSale[site.language]
-      : '';
+    const pointOfSale = resolvePointOfSale(site, language);
```

Comment:
If we have POS resolver, in this case, it needs to be customizable, so can be provided in config

Reply:
SiteResolver is more complex than PosResolver needs to be. I believe it's better to keep point of sale resolution simple, but separate, so that it can be easily changed if the point of sale data format changes in future.
But yes, we need to have a way to override resolution in middleware.